common/nexus/nexusrpc/completion.go

```diff
@@ -121,6 +127,10 @@ func (c *OperationCompletionSuccessful) applyToHTTPRequest(request *http.Request
 	}
 	request.Header.Set(headerOperationState, string(nexus.OperationStateSucceeded))
 
+	if c.CallbackToken != "" {
+		encodedToken := base64.StdEncoding.EncodeToString([]byte(c.CallbackToken))
+		request.Header.Set("Nexus-Callback-Token", encodedToken)
+	}
```

Comment:
```suggestion
	if c.CallbackToken != "" {
		encodedToken := base64.StdEncoding.EncodeToString([]byte(c.CallbackToken))
		request.Header.Set("Nexus-Callback-Token", encodedToken)
	}
```
this should likely be handled by the handler that receives the callback. should be `token` in completion requests